docs/content/latest/secure/audit-logging/audit-logging-ysql.md

```diff
@@ -202,36 +201,30 @@ Open the YSQL shell (ysqlsh), specifying the `yugabyte` user and prompting for t
     $ ./ysqlsh -U yugabyte -W
     ```
 
```

Comment:
Spacing and indenting problems, code blocks need 'sh' 'output etc

Reply:
fixed everywhere